README.md

```diff
@@ -17,6 +17,33 @@ composer require --dev spryker-sdk/integrator
 Available options:
 - `--format` - Define the format of the command output, example: json
 
+## Available commands
+
+### module:manifest:run
+
+Running the integrator in basic mode. Unapplied manifests will be downloaded from the [repo](https://github.com/spryker-sdk/integrator-manifests).
+The command expects not required argument `module-list`. With the argument, manifests will be applied only for modules that were specified.
+```
+integrator module:manifest:run <moduleNameA, moduleNameB (not required)>
+```
+
+
+### release-group:manifest:run
+
+Running the integrator for specific release group. Unapplied manifests will be downloaded from the S3 bucket.
```

Comment:
Why are they not inside the repo that we use? https://github.com/spryker-sdk/integrator-manifests

Reply:
According to the specification we have to use S3 storage https://spryker.atlassian.net/browse/SDK-1758

Reply:
@DmytroKlymanSpryker why are we switching here?
Afaik s3 was only supposed to be used for coverage diff

Reply:
I dont like that this is an own command.
Wouldnt it be cleaner to have an Option flag to switch the source?

Reply:
@dereuromark you are right, we use S3 only for generating coverage diff purposes (download manifests related to specific RG and store diff files)

I see two reasons, why better to have a separate command: 
* New command has own goal: generate and upload a diff file (instead of just applying manifests as the main command)
* Main command supports parameter modules (that means "apply manifests only for these modules") and we don't expect to support the parameter in the new command
* Main command checks that manifest was applied in the integrator.lock file - we don't expect this inside new command. 

Maybe we need to rename it to `release-group:diff:generate`

Reply:
Maybe `integrator:diff:generate` as we might not want to leak the internals (RG usage)?
cc @DmytroKlymanSpryker

Reply:
Thanks 
added changes